packages/patternlab/styleguide/source/_patterns/01-atoms/10-table/table-responsive.twig

```diff
@@ -6,21 +10,12 @@
 {% endif %}
 <div class="ma__table--responsive js-ma-responsive-table">
 
-  <nav class="ma__table__horizontal-nav">
-    <button class="ma__table__horizontal-nav__left" aria-controls="{{tableID}}">
-      <span class="visually-hidden">Scroll left</span>
-    </button>
-
-    <div class="ma__scroll-indicator" aria-controls="{{tableID}}" role="scrollbar" aria-orientation="horizontal">
-      <div class="ma__scroll-indicator__button"></div>
-    </div>
-
-    <button class="ma__table__horizontal-nav__right" aria-controls="{{tableID}}">
-      <span class="visually-hidden">Scroll right</span>
-    </button>
-  </nav>
-  <div class="ma__table--responsive__wrapper {{ mergedCells ? 'ma__table--merged-cells' : ''}}" id="{{tableID}}" tabindex="0" role="region" aria-orientation="horizontal" aria-label="{{table.description}}">
+  <div class="ma__table--responsive__wrapper {{ mergedCells ? 'ma__table--merged-cells' : ''}}" id="{{tableID}}" role="group" tabindex="-1">
     <table class="ma__table {{ tableWideClass }}">
+      <caption id="tbl-{{tableID}}" class="ma__table__caption">
+        {% if table.description %}<span class="ma__table__caption__content">{{ table.description }}</span>{% endif %}
+        <span class="ma__table__caption__scroll-info"> (Table in a horizontal scrolling container)</span>

```

Comment:
```suggestion
        <span class="ma__table__caption__scroll-info"> Note: Table has hidden columns, scroll horizontally to see more.</span>

```

I think we should use more user-friendly language. Maybe something like:
"Table has hidden columns, scroll horizontally to see more."